src/Input.java

```diff
@@ -0,0 +1,19 @@
+import java.util.Scanner;
+
+public class Input {
```

Comment:
I would consider naming this class `SpellingBeeCLI` just so that it's obvious that this is the main class. `Input` works but sounds a bit vague.